backend/src/schema/resolvers/reports.spec.js

```diff
@@ -20,7 +20,7 @@ describe('report mutation', () => {
 
   const action = () => {
     reportMutation = gql`
-      mutation($resourceId: ID!, $reasonCategory: String!, $reasonDescription: String!) {
+      mutation($resourceId: ID!, $reasonCategory: ReasonCategory!, $reasonDescription: String!) {
```

Comment:
Thanks that you have seen this and the related ones. 👍🏼 I was obviously in a hurry …
Sorry for that!

Reply:
no need to apologize, we got each others' backs